### ISSUE TITLE
Add Light calculations when converting to Indev.

### ISSUE THREAD

**Issue by VoxelBill** [VoxelBill/Classic-Converter#1]
Minecraft Classic doesn't store lighting so it will need to be calculated.

This is really a minor thing as currently converted indev worlds work fine but due to the ~3 seconds of the world being dark it causes some flowers to pop, and fixing the lighting should prevent that.

**Comment by Sl1mj1m14** [comment]
Actually in early indev light recalculation doesn't seem to occur at all, so essentially all grass converts to dirt and the world just stays dark. It might be worth looking into, or even potentially just patching it so the data for all blocks is set to 15

**Comment by BJTMastermind** [comment]
yeah setting all the air block spaces light value to 15 is probably what ill end up doing since it would be way easier.
